### PR TITLE
Add april 2021 newsletter

### PR DESCRIPTION
**Pull request by melcorr**
I am out tomorrow and Monday. 
If someone reviews this and finds I forgot some things, please comment and I'll fix on Tues. 
If someone reviews this and finds all is OK, feel free to publish tomorrow or Monday. 
Else, I'll publish on Tues. 

Thanks!

### CODE REVIEW

File: _posts/2021-04-29-foreman-community-newsletter-april-2021.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+<!--more-->
+
+
+## At long last, Katello 4.0 has arrived!
+
+This month, the Katello community reached a major milestone with the release of Katello 4.0. This release involved years of work, both on the [Pulp](https://pulpproject.org/) side, with the complete architectural overhaul that [led to the release of Pulp 3](https://opensource.com/article/20/8/manage-repositories-pulp), as well as massive changes on the Katello side, which were demoed at regular intervals over the last year or more. As well as this, a massive thank you is owed to everyone who asked questions and reported issues on the buildup to this major release.
+
+As well as the removal of Pulp 2, here are some of the major highlights of this release:
+
+* Katello repositories are now hosted at [yum.theforeman.org](yum.theforeman.org).
```
Comment (evgeni) [inline _posts/2021-04-29-foreman-community-newsletter-april-2021.md]:
```suggestion
* Katello repositories are now hosted at [yum.theforeman.org](https://yum.theforeman.org).
```

File: _posts/2021-04-29-foreman-community-newsletter-april-2021.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+* New Content View page user interface updates.
+* Content View Version added to Reporting Engine Template.
+* Number of Red Hat subscriptions consumed by host added to Entitlements report.
+* Added option to select Operating System in Register Content Host.
+
+For more information, check out the [release announcement](https://community.theforeman.org/t/katello-4-0-is-released/23245?u=mcorr).
+
+## Introduction to Katello
+
+On the topic of Katello, [@iballou](https://community.theforeman.org/u/iballou/summary) gave an introductory overview of Katello and its main features and workflows. You can rewatch the event on YouTube and if you have any questions, please post to our [Community forum](https://community.theforeman.org/).
```
Comment (evgeni) [inline _posts/2021-04-29-foreman-community-newsletter-april-2021.md]: Link to the YouTube video would be cool?

Edit: oh, it's embedded below, maybe reword to mention that fact? ;)

File: _posts/2021-04-29-foreman-community-newsletter-april-2021.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+## Host Details Redesign - Content Hosts
+
+Thank you to everyone who has contributed to efforts to gather feedback to help drive the host details redesign page. The next stage in the redesign process involves combining Content host and Hosts in one place. You can take a look at the [slidedeck from Red Hat Summit 2021](https://marvelapp.com/prototype/61fe028/screen/77569592) with example designs. Note that the UI mockups are skinned as Red Hat Satellite, but all of the UX considerations will affect Foreman users first. Also, [@marisvirik](https://community.theforeman.org/u/marisvirik/summary) will join us during the next Community Demo to talk about it. If you have any feedback, please complete the new [survey](https://docs.google.com/forms/d/e/1FAIpQLSc5pi28QYBXg4LZeuXT7ryP-ZFg6qqu2311Y0cfe9JyDIJ-jg/viewform).
+
+## Celebrating Foreman's Birthday
+
+The annual Foreman Birthday Party will take place on July 1st. Please save the date and [@Dirk](https://community.theforeman.org/u/dirk/) and his Netways teams will keep you up to date with the agenda and activities!
```
Comment (evgeni) [inline _posts/2021-04-29-foreman-community-newsletter-april-2021.md]:
```suggestion
The annual Foreman Birthday Party will take place on July 1st. Please save the date and [@Dirk](https://community.theforeman.org/u/dirk/) and his Netways team will keep you up to date with the agenda and activities! 
```

File: _posts/2021-04-29-foreman-community-newsletter-april-2021.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+__The focus of the Open Source Automation Days 2021 will be on Cloud, DevOps/GitOps, Containerization, Security Automation and Artificial Intelligence . Also of interest are exciting user stories about how how companies dealt with the subject of IT automation during the pandemic.__
+
+## Foreman Application Centric Deployment guide is now available!
+
+Application Centric Deployment (ACD) describes a fundamentally different approach to provisioning and configuring hosts in Foreman. Traditionally, Foreman has pursued a host-centric approach. Now, the ACD approach presents a new idea: deploying hosts to run an application that requires a predefined set of services. A [full user guide](https://docs.theforeman.org/nightly/Application_Centric_Deployment/index-foreman-el.html) for ACD is now available on the https://docs.theforeman.org. Let us know what you think!
```
Comment (evgeni) [inline _posts/2021-04-29-foreman-community-newsletter-april-2021.md]:
```suggestion
Application Centric Deployment (ACD) describes a fundamentally different approach to provisioning and configuring hosts in Foreman. Traditionally, Foreman has pursued a host-centric approach. Now, the ACD approach presents a new idea: deploying hosts to run an application that requires a predefined set of services. A [full user guide](https://docs.theforeman.org/nightly/Application_Centric_Deployment/index-foreman-el.html) for ACD is now available on [our docs site](https://docs.theforeman.org). Let us know what you think!
```

File: _posts/2021-04-29-foreman-community-newsletter-april-2021.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+This month, [@Marek_Hulan](https://community.theforeman.org/u/Marek_Hulan) posted a tutorial about how enable monitoring and controlling of the Puma service in a production environment on an RPM based distribution. Currently it’s not working for various reasons, but with newer versions of Puma, it should, but in the meantime, try this [workaround](https://community.theforeman.org/t/puma-status-monitoring-not-working-at-the-moment/23154).
+
+## Updates to the Foreman Contributor's Guide
+
+There have been a number of updates to the [Foreman contributor page](https://www.theforeman.org/contribute.html) that will hopefully make things easier if you would like to start contributing to the project.
+
+There are many ways to contribute. If you'd like to give back to the community in some way, but cannot contribute code, check out [the other types of contributions](https://www.theforeman.org/contribute.html#Othertypesofcontribution) section.  Contributions, both great and small, are very welcome.
+
+## Upgrading Postgresql on your Dev Setup
```
Comment (evgeni) [inline _posts/2021-04-29-foreman-community-newsletter-april-2021.md]:
```suggestion
## Upgrading PostgreSQL on your Dev Setup
```

**Review by evgeni — COMMENTED** [review]
Some minor comments

**Comment by melcorr** [comment]
Thanks @evgeni !
